docs/community/local-groups.mdx

```diff
@@ -64,6 +64,7 @@ us on [Discord](https://discord.com/invite/ktMAKGBnBs) to add your group.
 - [Antelope Valley Mesh](https://www.avmesh.org/)
 - [AltaMesh](https://altamesh.net/)
 - [Central Valley](https://centralvalleymesh.net/)
+- [Sac Valley Mesh](https://www.sacvalleymesh.com)
```

Comment:
when calling this URL with HTTPS i get a 404 error. HTTP works. Please change the link or update your hosting configuration.